### DISCUSSION TITLE
Infinity Loop Problem

### DISCUSSION THREAD

**Discussion by StepanGavrilov** [fkie-cad/cwe_checker#397] [Q&A]
Hello!

I have an infinity loop in this `forward_interprocedural_fixpoint.rs` in `lines 161-163`

when try to analyzer my example.

[libsqlite.so.zip](https://github.com/fkie-cad/cwe_checker/files/10941977/libsqlite.so.zip)

**Comment by Enkelmann** [comment]
This should be an issue (since it is probably a bug in the cwe_checker), so I opened issue #398 for it. Locking this discussion, we can move all further comments on it to issue #398.
